src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingLocations.java

```diff
@@ -5,17 +5,15 @@
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import uk.gov.hmcts.reform.sscs.validator.EnumPattern;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class HearingLocations {
```

Comment:
Could you please add the @JsonIgnoreProperties(ignoreUnknown = true) annotation?